### PR TITLE
Simple Weather in Vanilla JS

### PR DESCRIPTION
**Pull request by Ceejayski**
In this project, I created the following components:

- Store contains the local Storage components
- UI contains everything pertaining to UI and DOM
- Utils contains every function hitting the API(weather API, geolocation API)

Users can search for any city's weather and also change the measuring units.

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] You have mastered Git flow.
- [ ] PR has a descriptive title and summary
- [ ] README is professional and includes all the necessary info.

## To Improve 🙏
- [ ] README should be free of any grammar and spelling errors. Kindly run all its text through Grammarly to remove syntactic errors.
- [ ] App should change its appearance based on the received data. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png).
# Optional
- [ ] App fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location
### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes.
### Changes to make 🙅🏿 🙅🏿 🙅🏿 
- The image background should change based on the weather condition.
- Fix linter errors

### Martin Afani
#### Slack: whiz25

**Comment by Ceejayski** [comment]
> ## Changes Required 🙏🏿 🙏🏿 🙏🏿
> Nice work making the requested changes.
> 
> ### Changes to make 🙅🏿 🙅🏿 🙅🏿
> * The image background should change based on the weather condition.
> * Fix linter errors
> 
> ### Martin Afani
> #### Slack: whiz25

As requested by the First Reviewer through a screenshot saying " You should change the look of the page based on the data, maybe the color of the background or by adding images to describe the data" requesting for changes to the page not "necessarily" through the background but "maybe" through the background based on the weather which I did, if the next review walks through the code diligently he/she will see the basic steps carried out to do so, as I will damage the look of a good looking project to please a wrong review.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Ceejayski 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "rules": {
+    "no-shadow": "off",
+    "guard-for-in": "off",
+    "no-restricted-syntax": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/no-named-as-default": "off",
+    "import/no-named-as-default-member": "off",
+    "class-methods-use-this": "off"
+  },
```
Comment (mikenath223) [inline .eslintrc.json]: - [ ] Please ensure you use the same eslint rules as listed [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUESTED 🔴 ♻️ 

Hello there @Ceejayski 👋🏾 👋🏾 great job on working on the feedback left by the previous TSE. However, you have a failing eslint error, kindly fix it.


That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

**Comment by mimipeshy** [comment]
## STATUS: APPROVED 🥳 👯 
Hello team @atenaiis 👋🏾 👋🏾 congratulations on making it this far in this milestone 🥇 

#### Highlights

- You have worked on all changes requested by the previous TSE ✔️ 

With that, I approve of this project 😄 

Regards
__Peris Ndanu __
[GitHub ](https://github.com/mimipeshy)| [LinkedIn ](https://www.linkedin.com/in/peris-ndanu/)| [Twitter](https://twitter.com/pygirl254)

**Pull request merged**